types/setting.go

```diff
@@ -137,6 +137,7 @@ const (
 	SettingNameV2DataEngineFastReplicaRebuilding                        = SettingName("v2-data-engine-fast-replica-rebuilding")
 	SettingNameFreezeFilesystemForSnapshot                              = SettingName("freeze-filesystem-for-snapshot")
 	SettingNameAutoCleanupSnapshotWhenDeleteBackup                      = SettingName("auto-cleanup-when-delete-backup")
+	SettingNameAutoCleanupSnapshotAfterOnDemandBackupCompleted          = SettingName("auto-cleanup-after-on-demand-backup-completed")
```

Comment:
Just realized `auto-cleanup-when-delete-backup` misses `snapshot`...

```suggestion
	SettingNameAutoCleanupSnapshotAfterOnDemandBackupCompleted          = SettingName("auto-cleanup-snapshot-after-on-demand-backup-completed")
```

Reply:
>Just realized auto-cleanup-when-delete-backup misses snapshot...

Yes, it is.

```suggestion
	SettingNameAutoCleanupSnapshotAfterOnDemandBackupCompleted          = SettingName("auto-cleanup-snapshot-after-on-demand-backup-completed")
```
Modified. Thanks.